zipline/src/commonMain/kotlin/app/cash/zipline/internal/bridge/calls.kt

```diff
@@ -235,7 +237,7 @@ internal class ArgsListSerializer(
     encoder.encodeStructure(descriptor) {
       for (i in serializers.indices) {
         @Suppress("UNCHECKED_CAST") // We don't have a type argument T for each parameter.
-        encodeSerializableElement(descriptor, i, serializers[i] as KSerializer<Any?>, value[i])
+        encodeNullableSerializableElement(descriptor, i, serializers[i] as KSerializer<Any?>, value[i])
```

Comment:
This needs added to the README as an additional unstable API that we rely on. Is the `nullable` API stable? It may be worth the extra effort to reduce our reliance on unstable API in an otherwise stable library.

Reply:
> Is the `nullable` API stable?

Yep!

I'll have a go at it then in a follow up.

Reply:
Started a PR at https://github.com/cashapp/zipline/pull/1149